packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -134,62 +158,77 @@ export interface IBasePickerSuggestionsProps {
   * Function that specifies what to render when no results are found.
   */
   onRenderNoResultFound?: IRenderFunction<void>;
+
```

Comment:
Note that I have added blank lines between each prop. Please take care to keep the formatting consistent and clean.